provider/resource_keycloak_openid_client_optional_scopes_test.go

```diff
@@ -602,6 +605,7 @@ resource "keycloak_openid_client_optional_scopes" "optional_scopes" {
 		"address",
 		"phone",
 		"offline_access",
+                "microprofile-jwt",
```

Comment:
Can you fix the indentation for each instance of this string?

Reply:
Done

Reply:
Thanks!